src/app/zap-templates/zcl/data-model/chip/push-av-stream-transport-cluster.xml

```diff
@@ -0,0 +1,229 @@
+    <attribute code="0x0001" side="server" define="SUPPORTED_INGEST_METHODS" type="SupportedIngestMethodsBitmap" min="0x00" max="0x01">SupportedIngestMethods</attribute>
+    <attribute code="0x0002" side="server" define="CURRENT_CONNECTIONS" type="array" entryType="int16u">CurrentConnections</attribute>
+    <command code="0x00" source="client" name="AllocatePushTransport" optional="false" response="AllocatePushTransportResponse">
+      <description>This command SHALL allocate a transport and return a PushTransportConnectionID.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="TransportOptions" type="TransportOptionsStruct"/>
+    </command>
+
+    <command code="0x01" source="server" name="AllocatePushTransportResponse" optional="false" disableDefaultResponse="true">
+      <description>This command SHALL be generated in response to an AllocatePushTransport command.</description>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+      <arg id="1" name="TransportOptions" type="TransportOptionsStruct"/>
+      <arg id="2" name="TransportStatus" type="TransportStatusEnum" min="0x00" max="0x01"/>
+    </command>
+
+    <command code="0x02" source="client" name="DeallocatePushTransport" optional="false">
+      <description>This command SHALL be generated to request the Node deallocates the specified transport.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+    </command>
+
+    <command code="0x03" source="client" name="ModifyPushTransport" optional="false">
+      <description>This command is used to request the Node modifies the configuration of the specified push transport.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+      <arg id="1" name="TransportOptions" type="TransportOptionsStruct"/>
+    </command>
+
+    <command code="0x04" source="client" name="SetTransportStatus" optional="false">
+      <description>This command SHALL be generated to request the Node modifies the Transport Status of the transport.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="ConnectionID" type="int16u" isNullable="true"/>
+      <arg id="1" name="TransportStatus" type="TransportStatusEnum" min="0x00" max="0x01"/>
+    </command>
+
+    <command code="0x05" source="client" name="ManuallyTriggerTransport" optional="false">
+      <description>This command SHALL be generated to request the Node to manually start the specified push transport.</description>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+      <arg id="1" name="ActivationReason" type="TriggerActivationReasonEnum" isNullable="true" min="0x00" max="0x02"/>
+      <arg id="2" name="TimeControl" type="TransportMotionTriggerTimeControlStruct" optional="true" isNullable="true"/>
```

Comment:
The spec does not describe what null means for those fields.  Spec issue needed.

Reply:
Fixed by removing nullable